### DISCUSSION TITLE
GitHub action seems to be stuck...

### DISCUSSION THREAD

**Discussion by lhvy** [lowlighter/metrics#578] [General]
After merging #569, the workflow in the master branch seems to be stuck. https://github.com/lowlighter/metrics/runs/3902640411

The log won't load for me, so I'm not sure what's got it stuck.

**Comment by lhvy** [comment]
I just noticed the workflow was cancelled, not sure what is different between the copy in master and the workflow that passed in the PR...

**Comment by spenserblack** [comment]
Looks like it was re-run and succeeded https://github.com/lowlighter/metrics/runs/3906077067
